aries_cloudagent/commands/default_version_upgrade_config.yml

```diff
@@ -1,3 +1,5 @@
+v0.11.0:
+  update_routing_keys: true
```

Comment:
Assuming this is merged in for 0.11.0, I believe this is the correct way to specify that our upgrade routine should be run? @shaangill025